packages/ui-components/src/components/layout/Dialog.vue

```diff
@@ -214,6 +221,56 @@ const widthClasses = computed(() => {
   return classParts.join(' ')
 })
 
+const isFullscreenDesktop = computed(
+  () => props.fullscreen === 'desktop' || props.fullscreen === 'all'
+)
+
+const dialogPanelClasses = computed(() => {
+  const classParts: string[] = [
+    'transform md:rounded-xl text-foreground overflow-hidden transition-all bg-foundation text-left shadow-xl  flex flex-col md:h-auto'
+  ]
+
+  if (isFullscreenDesktop.value) {
+    classParts.push('md:h-full md:h-[98vh] md:!h-[98dvh]')
```

Comment:
Perfect! It looks a bit hacky, but i prefer this than using css

Reply:
It became very hard to read as inline within the template haha, I saw we used the pattern in other places so I switched it to this :)